src/MCPClient/tests/test_sanitize.py

```diff
@@ -128,3 +130,71 @@ def test_sanitize_object_names(self):
         finally:
             # Delete files
             shutil.rmtree(transfer_path)
+
+
+@pytest.mark.parametrize(
+    "basename, expected_name", [("helloworld", "helloworld"), (u"a\x80b", "ab")]
```

Comment:
Maybe throw in `Smörgåsbord.txt` as well, as a regression test for #352?

(I haven’t tested it with the old code; `a\x80b` is just what I happened to discover.)

Also I would double-check if my code does anything sensible with the case where all the characters get sanitised away, e.g. `\x80`. I’d want to test it doesn’t return an empty string in that case.

Reply:
So...it does return an empty string, but it did before your changes (`unidecode(u"\x80") == ""`). I'll put a check in for that case.